.jenkins/pytorch/win-test-helpers/installation-helpers/install_magma.bat

```diff
@@ -1,6 +1,8 @@
 rem remove dot in cuda_version, fox example 11.1 to 111
 set VERSION_SUFFIX=%CUDA_VERSION:.=%
-set CUDA_SUFFIX=cuda%VERSION_SUFFIX%
+IF NOT "%VERSION_SUFFIX%"=="cpu" (
+    set CUDA_SUFFIX=cuda%VERSION_SUFFIX%
+)
```

Comment:
Why this is needed? I.e. this script shouldn't at all be called for CPU builds, should it?

Reply:
I didn't know 16.8 is unsupported. My local machine VS version is 16.8.5 by chance and it is built successfully. 
I haven't find official documents for the relation between the VS version and VC version. The VC version number is found locally.

For install_magma.bat, I find the VERSION_SUFFIX become cpu when the vc version isn't correct. for example, we can see it in the log of https://github.com/pytorch/pytorch/pull/54399. So I added it for safety. I think it could be removed.

Reply:
I like the check, but perhaps we can throw an error there if VERSION_SUFFIX is set to cpu?